between/between_test.go

```diff
@@ -143,9 +143,16 @@ var _ = DescribeTable("Between", (testCase).Run,
 	}),
 
 	Entry("between the start and stop time but the compare time is in a different timezone", testCase{
-		start:         "2:00 AM -0600",
-		stop:          "6:00 AM -0600",
-		timeToCompare: "1:00 AM -0700",
+		start:         "00:01 AM -0700",
```

Comment:
Did this change cause the test to fail before the code change was made? I think the example times were clearer before.